docs/aspell-dictionary.pws

```diff
@@ -0,0 +1,432 @@
+personal_ws-1.1 en 431
+ABI
+ADDI
+API
+APIs
+AnotherVecType
+BFLOAT
+BNT
+BRISC
+BRISCs
+BUF
+BUFs
+BinaryOpType
+BorrowedStorage
+BufferType
+CBHandle
+CBs
+CLI
+CMD
+CPUs
+CRBW
+CSV
+CWFW
+CircularBufferConfig
+CircularBuffers
+CloseDevice
+CommandQueue
+CompileProgram
+ComputeConfig
+ConfigureDeviceWithProgram
+CoreCoord
+CoreRange
+CoreRangeSet
+CreateBuffer
+CreateCircularBuffer
+CreateComputeKernel
+CreateDataMovementKernel
+CreateDevice
+CreateKernel
+CreateProgram
+CreateSemaphore
+DEFAULTFLOAT
+DPRINT
+DataFormat
+DataMovementConfig
+DataMovementProcessor
+DataType
+DeallocateBuffer
+DeviceStorage
+DstMode
+DumpDeviceProfileResults
+ENDL
+EltwiseUnary
+EnqueueProgram
+EnqueueReadBuffer
+EnqueueWriteBuffer
+Enum
+Enums
+FW
+FileCopyrightText
+GGGG
+GetCircularBufferConfig
+GetNumAvailableDevices
+GetNumPCIeDevices
+GetRuntimeArgs
+Grayskull
+HW
+HiFi
+InterleavedBufferConfig
+Jupyter
+KernelHandle
+KtNt
+LREG
+LREGs
+LReg
+LRegs
+LaunchProgram
+LayoutConversionOnHost
+LoFi
+MULI
+Makefile
+MatMul
+MathFidelity
+MemoryConfig
+Metalium
+MtKt
+MtNt
+NCRISC
+NCRISCs
+NOC
+NOP
+Namespace
+NewOperation
+NoC
+NoCs
+Nt
+OPs
+OwnedStorage
+PCC
+PCI
+PCIe
+POPC
+PUSHC
+PYTHONPATH
+ParallelizationStrategyEnum
+PoolType
+Profiler
+ProgramWithCallbacks
+Pytorch
+QueryDevices
+README
+RISCV
+RISCVs
+RISCs
+ReadFromDevice
+ReduceDim
+ResNet
+SETPRECISION
+SETW
+SFP
+SFPI
+SFPOR
+SFPREPLAY
+SFPU's
+SIMD
+SPDX
+SetRuntimeArgs
+ShardedBufferConfig
+SliceRange
+TRISC
+TSLICE
+TTNN
+TTNN's
+TensorMemoryLayout
+Tensorflow
+Tenstorrent
+Theano
+TileSize
+TileSlice
+Tilizing
+UnaryOpType
+UpdateCircularBufferPageSize
+UpdateCircularBufferTotalSize
+UpdateDynamicCircularBufferAddress
+VC
+VCs
+WH
+WR
+WriteToBuffer
+WriteToDevice
+XXXX
+ZoneScoped
+abovementioned
+acc
+acos
+addexp
+addr
+agrebenisan
+api
+apis
+arg
+args
+asin
+asm
+async
+atan
+autoclass
+autofunction
+backend
+bcast
+behaviour
+bfloat
+bmm
+bool
+boolean
+breakpoint
+brisc
+cachable
+cb
+cbid
+cd
+cin
+ckernel
+ckernels
+cli
+codebase
+concat
+config
+const
+constexpr
+coord
+coords
+cpp
+cpu
+cq
+cref
+cstdint
+csv
+ctl
+datacopy
+dataflow
+datatypes
+deallocate
+debiases
+dec
+defaultfloat
+denorms
+dest
+dev
+devs
+dfb
+dir
+dnn
+doxygendefine
+doxygenfunction
+dprint
+dst
+dt
+dtype
+durations
+elementwise
+elseif
+eltwise
+elu
+endblock
+endif
+enqueue
+enqueued
+enum
+env
+eqz
+erf
+erfc
+erfinv
+exexp
+exman
+expm
+fb
+fno
+fp
+funcs
+gcc
+gdb
+gdb's
+gelu
+genindex
+gez
+grayskull
+grey
+gtz
+hardsigmoid
+hardswish
+heaviside
+hpp
+hs
+html
+hw
+icb
+idst
+ie
+ifrom
+implict
```

Comment:
```
implict
```

Looks I missed some more. Sorry about that. We want to get this right.

Since it's so light, let's make sure we scrutinize as much as we can as feedback is quick via CI.